package.json

```diff
@@ -20,8 +20,7 @@
   },
   "files": [
     "dist/*.{js,map}",
-    "bs-breakpoints.d.ts",
-    "CHANGELOG.md"
```

Comment:
CHANGELOG.md is added by default ?

Reply:
Yeah, https://docs.npmjs.com/files/package.json#files